LiquidBounce/settings/redesky

```diff
@@ -436,28 +378,44 @@ Tower JumpMotion 0.41
 Tower JumpDelay 0
 Tower ConstantMotion 0.42
 Tower ConstantMotionJumpGround 1.0
+Tower TeleportHeight 1.0
+Tower TeleportDelay 0
+Tower TeleportGround true
+Tower TeleportNoMotion false
 Tower Counter true
-
 Trigger MaxCPS 12
 Trigger MinCPS 8
-
 Velocity Horizontal 0.0
 Velocity Vertical 0.0
 Velocity Mode Simple
-
+Velocity ReverseStrength 1.0
+Velocity SmoothReverseStrength 0.02
+Velocity AACPushXZReducer 2.0
+Velocity AACPushYReducer true
 WallClimb Mode Clip
 WallClimb ClipMode Fast
 WallClimb CheckerClimbMotion 0.0
-
 WaterSpeed Speed 1.5
-
 Zoot BadEffects true
 Zoot Fire true
 Zoot NoAir false
 
-#Chat
-chat &4&lDescriptions: 
-chat &6&l=====================================================
-chat &6Hypixel Fly, Longjump and ExpandScaffold (6) are working.
-chat &5&lSettings made by Sieoux &f| &5Cleared Settings Text by &bSpoilerRules&f.
-chat &4&lSettings Uploaded by Ghost: http://yt.ghostyt.de
+# Toggles
+Criticals toggle true
+NoSlow toggle true
+AutoArmor toggle true
+InventoryCleaner toggle true
+Step toggle true
+ChestStealer toggle true
+InventoryMove toggle true
+Velocity toggle true
+Fullbright toggle true
+LongJump toggle false
+FastClimb toggle false
+BugUp toggle false
+
+# Chat-msg
+chat &r
+chat &c&lREDESKY.COM CONFIG &7| &b&lUPDATED: 15/7/2020
+chat &cSet your &c&lTargets&c to only 'Players'!
```

Comment:
you can add
`targetPlayer true`
instead of chat thing

Reply:
i realised that after i submit the request lol

Reply:
but yeah true^^^